src/Corax/Querying/IndexSearcher.Search.cs

```diff
@@ -90,7 +96,35 @@ public IQueryMatch SearchQuery(FieldMetadata field, IEnumerable<string> values,
                     Constants.Search.Operator.And => And<IQueryMatch, MultiTermMatch>(searchQuery, query, token: cancellationToken),
                     _ => throw new ArgumentOutOfRangeException(nameof(@operator), @operator, null)
                 };
+                
+                continue;
+            }
+            
+            //Phrase query
+            var startPosition = terms.Count;
```

Comment:
This should always be zero, no?
Above, we call `terms.Clear()` before `EncodeAndApplyAnalyzerForMultipleTerms`, why not here too?